### PR TITLE
[FEATURE] allow exclude definition global

### PR DESCRIPTION
**Pull request by bennsel**
Hi,

I've allowed the exclude definition to be globally defined. My Configuration uses on every environment the same excludes and so i must implement duplicate code. With this feature i define the excludes only one time. 

Now its both possible global and environment specific

Example: tests/Resources/global-exclude.yml

### CODE REVIEW

**Comment by andres-montanez** [comment]
Hi, thanks! Looks pretty handy, I will review it and schedule it for v3.2.
Cheers!

File: tests/Command/BuiltIn/DeployCommandMiscTasksTest.php [hunk 1]
```diff
@@ -50,9 +50,27 @@ public function testSymfonyEnvironmentConfiguration()
         $this->assertEquals(0, $tester->getStatusCode());
     }
 
-    public function testComposerFlags()
+    public function testComposerFlagsDataProvider() {
```
Comment (thePanz) [inline tests/Command/BuiltIn/DeployCommandMiscTasksTest.php]: Non-test functions should not start with "test" prefix :)

Comment (bennsel) [inline tests/Command/BuiltIn/DeployCommandMiscTasksTest.php]: totally right. i've correct it

**Comment by andres-montanez** [comment]
Hi, thanks for the PR. I've merged but then noted you have done it to the `master` branch, so I had to revert it and do it by hand to the correct branch which is `nostromo`.

So, thanks for the PR and the improvement but for further PRs please do it always to the `nostromo` branch.

Regards.

**Pull request merged**